canonical-web-and-design/maas-ui
# feat(ui): add the machine details numa card

## Done

- Add a numa card to the summary tab of the machine details.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the machine details for a machine with a numa node.
- You should see a numa card with the details.
- Try with greater than and less than 2 numa nodes and you should see expanded/expandable versions of the card.

## Fixes

Fixes: canonical-web-and-design/MAAS-squad#2110
